### ISSUE TITLE
Webpack build doesn't substitute "__VERSION__"

### ISSUE THREAD

**Issue by mrlika** [Novage/p2p-media-loader#5]
Unlike browserify build that substitutes "__VERSION__" string with package version using browserify-versionify plugin.

**Comment by kslimani** [comment]
@mrlika in your `webpackfile.js` you could add to `plugins` configuration attribute something similar to :

```javascript
const version = require('./package.json').version

function makeConfig({libName, entry, mode}) {
    return {
        // [...]
        plugins: [
            new webpack.DefinePlugin({__VERSION__: JSON.stringify(version)})
        ]
    }
};
```

Note: you may needs to remove `__VERSION__` quotes in source code. _(will be added because of stringify)_

**Comment by mrlika** [comment]
Fixed in 67c35c78220c3f042ba2eb40434be1404d0a540d
